app/src/services/market_maker.ts

```diff
@@ -85,8 +94,12 @@ class MarketMakerService {
     const { balanceOfForYes, balanceOfForNo } = balanceInformation
 
     const totalBalance = balanceOfForYes.add(balanceOfForNo)
-    const actualPriceForYes = divBN(balanceOfForNo, totalBalance)
-    const actualPriceForNo = divBN(balanceOfForYes, totalBalance)
+    const actualPriceForYes = !totalBalance.isZero()
+      ? divBN(balanceOfForNo, totalBalance)
+      : balanceOfForNo.toNumber()
+    const actualPriceForNo = !totalBalance.isZero()
+      ? divBN(balanceOfForYes, totalBalance)
+      : balanceOfForYes.toNumber()
```

Comment:
@fvictorio Please take a look at this carefully

Reply:
The only way `totalBalance` can be zero is if both `balanceForYes` and `balanceForNo` are zero (or one of them is negative, which shouldn't happen). I believe this only happens if the market has no funding. This situation should probably be handled in some other way. But, as a quick fix, I think this fix is OK (maybe we could just use 0 instead of `balanceOfFor{Yes,No}.toNumber()` to make it less obscure).

Reply:
I will use 0 instead of `balanceOfFor...`